Makefile

```diff
@@ -20,30 +31,36 @@ ifeq ($(PARENT_BRANCH_SUFFIX), main)
 CERT_MANAGER_OPERATOR_BRANCH = master
 endif
 
-## default container build tool to use for creating images.
-CONTAINER_ENGINE ?= docker
+## container build tool to use for creating images.
+CONTAINER_ENGINE ?= podman
 
-## default image name for cert-manager-operator.
+## image name for cert-manager-operator.
 CERT_MANAGER_OPERATOR_IMAGE ?= cert-manager-operator
 
-## default image name for cert-manager-operator-bundle.
+## image name for cert-manager-operator-bundle.
 CERT_MANAGER_OPERATOR_BUNDLE_IMAGE ?= cert-manager-operator-bundle
 
-## default image name for cert-manager.
+## image name for cert-manager.
 CERT_MANAGER_IMAGE ?= cert-manager
 
-## default image name for cert-manager-acmesolver.
+## image name for cert-manager-acmesolver.
 CERT_MANAGER_ACMESOLVER_IMAGE ?= cert-manager-acmesolver
 
-## default image version to tag the created images.
-IMAGE_VERSION ?= release-$(PARENT_BRANCH_SUFFIX)
+## image version to tag the created images with.
+IMAGE_VERSION ?= $(release_version)
 
-## default image tag makes use of the branch name and
+## image tag makes use of the branch name and
 ## when branch name is `main` use `latest` as the tag.
 ifeq ($(PARENT_BRANCH_SUFFIX), main)
 IMAGE_VERSION = latest
 endif
 
+## args to pass during image build
+IMAGE_BUILD_ARGS ?= --build-arg RELEASE_VERSION=$(release_version) --build-arg COMMIT_SHA=$(commit_sha) --build-arg SOURCE_URL=$(source_url)
+
+## tailored command to build images.
+IMAGE_BUILD_CMD = $(CONTAINER_ENGINE) build $(IMAGE_BUILD_ARGS)
```

Comment:
We cannot use common `IMAGE_BUILD_ARGS` for both operator and operand container image builds, because versions will differ.

For eg. today our bundle contains cert-manager-operator v1.15.0 based out of (jetstack-)cert-manager v1.15.2.

Reply:
This is for local build and to test and verify the image building. The version passed as arg doesn't contain minor version.

Reply:
That's fine, but we'd want to keep it as realistic as possible.
What release version-ing is used today on the Konflux builds for the ones on the PR and without?

Reply:
I mean, how is the value of `${RELEASE_VERSION}` in the Containerfile(s) during Konflux builds.

Reply:
It's initialized [here](https://github.com/openshift/cert-manager-operator-release/blob/release-1.15/.tekton/cert-manager-operator-1-15-pull-request.yaml#L44)